articles/per_package_documentation.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Per-Package Documentation
+permalink: articles/per_package_documentation.html
+abstract: This article describes the requirements and design of ROS 2’s per-package documentation system.
+author: Marya Belanger
+published: true
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Background
+
+ROS 2 is lacking a process for the conglomeration of package documentation.
+Discoverability for package documentation (and documentation in general) is one of the most frequent user complaints.
+
+With a standardized package documentation process that encourages package maintainers to robustly document their packages, and presents the results alongside ROS 2's general documentation, issues of discoverability and consistency will be addressed.
```

Comment:
I'm having trouble parsing this sentence.

Reply:
Maybe something like:

```
The goal of this design is to standardize a documentation process for ROS 2 packages to encourage maintainers to robustly document their packages.  The results will be presented in a consistent way alongside the general ROS 2 documentation, making the documentation as a whole discoverable and consistent.
```